### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by evanharmon1**
Pretty minor typo, spacing, and various pythonic things according to my understanding if you want to even bother merging them. Your comments are very helpful. I'll try to keep a lookout for adding other explanatory comments as I'm getting more familiar with the code.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -3,37 +3,38 @@
...
+        # TODO: make this a property instead of a getter method
         return [c.name for c in cls().__table__.columns]
 
+
 # globally accessible database connection
 db = SQLAlchemy(model_class=BaseModel)
 
 # for some reason doing this import makes models accessible to create_app
 # TODO: see if app can be instantiated without this import
 from models import User, Role, Project, ProjectType, Location
```
Comment (t-ketchem) [inline app.py:29]: The master branch doesn't have this... I'm not sure it is needed

**Review by t-ketchem — APPROVED** [review]
I don't see any reason not to merge this ASAP

File: app.py (hunk 1 above)
Comment (galbwe) [reply app.py:29]: @t-ketchem do you mean lines 27-29? I agree that they are not necessary. I don't remember why I originally included those imports.

```
# for some reason doing this import makes models accessible to create_app
# TODO: see if app can be instantiated without this import
from models import User, Role, Project, ProjectType, Location
```

**Comment by evanharmon1** [comment]
So this should close issue #8 ? Correct?

**Review by galbwe — APPROVED** [review]
Thanks. I didn't realize I had been doing docstrings wrong this whole time.

**Pull request merged**